doc/source/whatsnew/v0.21.1.txt

```diff
@@ -117,7 +117,8 @@ Categorical
 ^^^^^^^^^^^
 
 - Bug in :meth:`DataFrame.astype` where casting to 'category' on an empty ``DataFrame`` causes a segmentation fault (:issue:`18004`)
--
+- Error messages in the testing module have been improved when items have
+  different CategoricalDtype (:issue:`18069`)
```

Comment:
Can you add backticks to <code>\`\`CategoricalDtype \`\`</code>?  Looks like that's what was typically done in the previous whatsnew.